com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -106,6 +112,12 @@ void InternalLoadResource(VisualEffectResource resource)
             graphView.controller = VFXViewController.GetController(resource, true);
             graphView.UpdateGlobalSelection();
             graphView.FrameNewController();
+            graphView.UpdateIsSubgraph();
+        }
+
+        public bool CanPopResource()
```

Comment:
I introduced a "back" button when we entered a subgraph, this is to know if the button should be displayed or not